composer-7.4.lock

```diff
@@ -0,0 +1,3813 @@
+{
+    "_readme": [
+        "This file locks the dependencies of your project to a known state",
+        "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
+        "This file is @generated automatically"
```

Comment:
Where's the "@" come from?

I won't look at this file in detail assuming it's `@generated`.